stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>Jennifer Ding:</p><ul><li>The use of na_values='.' was new to me and an interesting approach to surface null values. That could be worth explaining</li></ul><p>Joe Cerniglia:</p><p>Until you mentioned this, I believed the na_values option to be a way to make null values uniform. Now I can see the option is more refined than that. The na_values is meant to recognize (surface, as you say) additional strings as null, in addition to a lengthy list of default values that are supplied. From the UCI Machine Learning Repository, I can see from the general specifications of the glass dataset that it contains no nulls; therefore, the option is not necessary. I would recommend for clarity that the na_values option be removed from the function call. I can include an explanation in the text of why it was not used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>